filer/locale/bg/LC_MESSAGES/django.po

```diff
@@ -3,13 +3,14 @@
 # This file is distributed under the same license as the PACKAGE package.
 # 
 # Translators:
+# Translators:
 msgid ""
 msgstr ""
-"Project-Id-Version: django-filer\n"
+"Project-Id-Version: django Filer\n"
```

Comment:
@jmelett only the labels on Transifex:
![image](https://cloud.githubusercontent.com/assets/270307/16195771/753cbf06-36fb-11e6-820a-021129ba3d65.png)